vllm/model_executor/layers/attention.py

```diff
@@ -355,7 +355,14 @@ def forward(
 
         # Apply rotary embedding to the query and key before passing them
         # to the attention op.
-        query, key = self.rotary_emb(positions, query, key)
+        if not isinstance(self.rotary_emb,
+                          DynamicNTKScalingRotaryEmbeddingQwen):
+            query, key = self.rotary_emb(positions, query, key)
+        else:
+            # need to pass in prompt_token_length to compute dynamic_ntk for qwen
+            query, key = self.rotary_emb(
+                positions, query, key,
+                input_metadata.seq_groups[0][1].prompt_token_length)
```

Comment:
In high concurrency scenarios, the model may still output invalid information or even no output. I think this is key problem.